Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml

```diff
@@ -53,6 +53,55 @@ tasks:
     scriptarguments:
       owners:
         simple: ${alert.asmserviceownerunrankedraw}
+      asmsystemids:
+        complex:
+          root: alert.asmsystemids
+          filters:
+          - - operator: containsString
+              left:
+                value:
+                  simple: alert.asmsystemids.type
+                iscontext: true
+              right:
+                value:
+                  simple: ASSET-NAME
+          transformers:
+          - operator: getField
+            args:
+              field:
+                value:
+                  simple: id
+          filters:
```

Comment:
@BigEasyJ don't we actually want to remove lines 74-104?

Reply:
i went ahead and made that change; please review [this commit](https://github.com/demisto/content/pull/28375/commits/f050426e4f9ac760ceeaae503db74b934def8c72)